src/scripts/main.js

```diff
@@ -1,3 +1,425 @@
+/* eslint-disable max-len */
 'use strict';
 
-// write your code here
+// const { formatters } = require('stylelint');
+
+const start = document.querySelector('.start');
+const restart = document.querySelector('.restart');
+const startMessage = document.querySelector('.message-start');
+const gameScore = document.querySelector('.game-score');
+const info = document.querySelector('.info');
+const winMessage = document.querySelector('.message-win');
+const gameOverMessage = document.querySelector('.message-lose');
+const table = document.querySelector('table');
+const [...rowsArray] = table.rows;
+
+const typeOfMoves = {
+  ArrowLeft: 'leftHandler',
+  ArrowRight: 'rightHandler',
+  ArrowUp: 'upHandler',
+  ArrowDown: 'downHandler',
+};
+const cellStyle = new Map([
+  [2, 'field-cell--2'],
+  [4, 'field-cell--4'],
+  [8, 'field-cell--8'],
+  [16, 'field-cell--16'],
+  [32, 'field-cell--32'],
+  [64, 'field-cell--64'],
+  [128, 'field-cell--128'],
+  [256, 'field-cell--256'],
+  [512, 'field-cell--512'],
+  [1024, 'field-cell--1024'],
+  [2048, 'field-cell--2048'],
+]);
+
+class Tile {
+  constructor() {
+    this.isOccupied = false;
+    this.className = cellStyle.get(0);
+    this.value = 0;
+    this.row = 0;
+    this.cell = 0;
+    this.td = null;
+
+    this.currentPosition = {
+      x: 0,
+      y: 0,
+    };
+  }
+}
+class Game {
+  constructor() {
+    this.message = null;
+    this.startCellsNum = 0;
+    this.height = 4;
+    this.width = 4;
+    this.field = [];
+    this.gameScore = 0;
+    this.availableCells = 16;
+    this.winningNumber = 2048;
+    this.bonusScoreNode = null;
+    this.bonusScore = 0;
+  }
+  initField(arr, width) {
```

Comment:
```suggestion
  }
  
  initField(arr, width) {
```
добавляй пробелы между методами